packages/components/src/app/word-query/word-query-controller.js

```diff
@@ -80,15 +80,15 @@ export default class WordQueryController {
     const watchableQuery = this._apolloClient.watchQuery({ query: WORD_QUERY, pollInterval, variables })
     watchableQuery.subscribe({
       next: ({ data }) => {
-        const loading = data.word.state.loading
+        const loading = data.word.extensions.state.loading
         if (loading === false) {
           watchableQuery.stopPolling()
         }
         if (counter >= maxIterations) {
           watchableQuery.stopPolling()
         }
         counter++
-        dataCallback(data.word.homonyms, data.word.state, data.word.errors)
+        dataCallback(data.word)
```

Comment:
I think it's better to pass the whole response the the calling function rather than parse it here: it might know better how to deal with the data returned.